packages/firestore/src/local/simple_db.ts

```diff
@@ -162,10 +162,20 @@ export class SimpleDb {
     // ua = 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML,
     // like Gecko) Chrome/39.0.2171.71 Safari/537.36 Edge/12.0';
 
+    // iOS Safari: Disable for users running iOS version < 10.
+    const iOSVersion = SimpleDb.getIOSVersion(ua);
+    const isUnsupportedIOS = 0 < iOSVersion && iOSVersion < 10;
+
+    // Android browser: Disable for userse running version < 4.4.
+    const androidVersion = SimpleDb.getAndroidVersion(ua);
+    const isUnsupportedAndroid = 0 < androidVersion && androidVersion < 4.4;
```

Comment:
Can't this be caught by feature detection?  https://caniuse.com/#feat=indexeddb shows there is no support for indexed db for android version < 4.4